docs/reference-guides/interactivity-api/api-reference.md

```diff
@@ -1054,7 +1054,12 @@ Apart from the store function, there are also some methods that allows the devel
 
 #### getContext()
 
-Retrieves the context inherited by the element evaluating a function from the store. The returned value depends on the element and the namespace where the function calling `getContext()` exists.
+Retrieves the context inherited by the element evaluating a function from the store. The returned value depends on the element and the namespace where the function calling `getContext()` exists. It can also take an optional namespace argument to retrieve the context of a specific interactive region.
+
+```js
+const context = getContext(namespace);
```

Comment:
```suggestion
const context = getContext('namespace');
```

I would suggest still wrapping the namespace here in quotes just to indicate that it is a string